client/flutter/lib/pages/travel_advice.dart

```diff
@@ -7,17 +7,26 @@ import 'package:url_launcher/url_launcher.dart';
 class TravelAdvice extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    var travelAdviceContainerTextArray=S.of(context).travelAdviceContainerText.split('*');
     return PageScaffold(context,
     showShareBottomBar: false,
         body: [
           SliverList(
               delegate: SliverChildListDelegate([
             Container(
               color: Color(0xffD82037),
-              padding: EdgeInsets.symmetric(horizontal: 25, vertical: 10),
-              child: Text(
-                S.of(context).travelAdviceContainerText,
-                style: TextStyle(color: Colors.white, fontSize: 16),
+              padding: EdgeInsets.symmetric(horizontal: 25, vertical: 25),
+              child: RichText(
+                text: TextSpan(
+                  style: TextStyle(color: Colors.white,fontSize: 16),
+                  children: <TextSpan>[
+                    TextSpan(text: travelAdviceContainerTextArray[0]),
```

Comment:
This might no be the same order the bold text is displayed in other languages. I would make something more generic that not depends on only the number of * or their position. 

In other parts of the project [question_index.dart](https://github.com/WorldHealthOrganization/app/blob/master/client/flutter/lib/pages/question_index.dart) html tags are being used for styling text

Reply:
Okay, understood. Thanks.